contracts/hydro/src/utils.rs

```diff
@@ -543,3 +545,62 @@ pub struct LockingInfo {
     pub lock_in_public_cap: Option<u128>,
     pub lock_in_known_users_cap: Option<u128>,
 }
+
+pub fn has_nonzero_funds(liquidity_deployment: LiquidityDeployment) -> bool {
```

Comment:
Nitpick: this could be implemented as a method of the `LiquidityDeployment` struct.